src/main/java/com/example/spotikservice/service/SpotifyService.java

```diff
@@ -1,18 +1,24 @@
 package com.example.spotikservice.service;
 
-import se.michaelthelin.spotify.model_objects.specification.AlbumSimplified;
-import se.michaelthelin.spotify.model_objects.specification.PlaylistSimplified;
-import se.michaelthelin.spotify.model_objects.specification.PlaylistTrack;
+import se.michaelthelin.spotify.model_objects.specification.*;
 
 import java.util.List;
-import java.util.Map;
+import java.util.TreeMap;
 
 public interface SpotifyService {
-    PlaylistSimplified[] getPlaylists();
+    List<PlaylistSimplified> getPlaylists();
 
-    Map<String, List<AlbumSimplified>> getLastReleasesFromSubscribedArtists();
+    List<Album> getAlbums();
 
-    List<PlaylistTrack> getRussianTracks(String id);
+    List<Track> getSavedSongs();
+
+    List<TrackSimplified> getRussianTracksFromAlbum(String albumId);
```

Comment:
Can you change the logic a little bit? Let's pass a country code (country name) as a parameter. In that case, we could use this method for all the countries we wanted